### PR TITLE
Elasticsearch: Change feature toggle for backend migration to enableElasticsearchBackendQuerying

### PR DESCRIPTION
**Pull request by ivanahuckova**
This PR changes `disableElasticsearchBackendQuerying` -> `enableElasticsearchBackendQuerying` to default to not running queries trough backend and only enabling it when users set `enableElasticsearchBackendQuerying` to `true`. 

Part of #54011

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md [hunk 1]
```diff
@@ -71,7 +71,6 @@ Alpha features might be changed or removed without prior notice.
 | `storage`                          | Configurable storage for dashboards, datasources, and resources                                                                                                                                     |
 | `exploreMixedDatasource`           | Enable mixed datasource in Explore                                                                                                                                                                  |
 | `newTraceView`                     | Shows the new trace view design                                                                                                                                                                     |
-| `correlations`                     | Correlations page                                                                                                                                                                                   |
```
Comment (ivanahuckova) [inline docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:74]: This change is not related and will be removed when https://github.com/grafana/grafana/pull/66499 is merged.

**Review by matyax — APPROVED** [review]

**Review by gabor — APPROVED** [review]
LGTM!
